tests/common/devices.py

```diff
@@ -268,6 +268,21 @@ def _get_platform_info(self):
                 result["hwsku"] = line.split(":")[1].strip()
             elif line.startswith("ASIC:"):
                 result["asic_type"] = line.split(":")[1].strip()
+
+        if result["platform"]:
+            platform_file_path = os.path.join('/usr/share/sonic/device', result["platform"], 'platform.json')
+
+            try:
+                out = self.command("cat {}".format(platform_file_path))
+                platform_info = json.loads(out["stdout"])
+                result["chassis"] = platform_info["chassis"]
+            except:
+                # if platform.json does not exist, then it's not added currently for certain platforms
+                # eventually all the platforms should have the platform.json
+                logging.debug("platform.json is not available for this platform, dut facts will not contain
+                              chassis and interfaces data")
```

Comment:
As with the above comment, this message should be more generic, and not specify "chassis and interfaces data". Suggest rewording along the lines of,

```
"platform.json is not available for this platform. DUT facts will not contain complete platform information."
```

Reply:
fixed